synapse/third_party_rules/access_rules.py

```diff
@@ -557,11 +561,12 @@ def _get_members_and_tokens_from_state(state_events):
         """
         existing_members = []
         threepid_invite_tokens = []
-        for key, event in state_events.items():
-            if key[0] == EventTypes.Member:
-                existing_members.append(event.state_key)
-            if key[0] == EventTypes.ThirdPartyInvite:
-                threepid_invite_tokens.append(event.state_key)
+        for key, state_event in state_events.items():
```

Comment:
I'm not sure this is a worthwhile improvement

Reply:
The reason I kept it is that I tried modifying the function by having it process the current event we need to validate and lost a lot of time trying to find out why `event` wasn't giving me the right value. I believe more precise variable names can't hurt, and this one might prevent something like that to happening in the future if we need to modify this function.

Reply:
fair enough